client/lib/analytics/index.js

```diff
@@ -37,7 +38,61 @@ window.ga = window.ga || function() {
 };
 window.ga.l = +new Date();
 
-loadScript( '//stats.wp.com/w.js?56' ); // W_JS_VER
+function getUrlParameter( name ) {
+	name = name.replace( /[\[]/, '\\[' ).replace( /[\]]/, '\\]' );
+	const regex = new RegExp( '[\\?&]' + name + '=([^&#]*)' );
+	const results = regex.exec( location.search );
+	return results === null ? '' : decodeURIComponent( results[ 1 ].replace( /\+/g, ' ' ) );
+}
```

Comment:
it would be great if we could explain what's happening in this function and why. It's not at all obvious why this function exists, other than it appears to have something to do with URL query strings. Also, we seem to be replacing brackets with some kind of escaped brackets, but I don't see this function ever called with a `name` having brackets! What's up with that?

Using more specific names (and dropping the mutation on line 42) could help that clarity. A JSDoc block explaining the purpose is a must.